extensions/wikia/Search/css/WikiaSearch.scss

```diff
@@ -41,14 +45,40 @@ $form-negative-side-padding: $form-side-padding*-1;
 		background: $color-search-form-background;
 		border-top: 1px solid $color-search-grid-border;
 		border-bottom: 1px solid $color-search-grid-border;
-		overflow: hidden;
 		padding: 9px 10px;
 
 		.inter-wiki-search & {
 			margin-top: 10px;
 			padding-bottom: 29px;
 		}
 
+		.wds-dropdown {
+			background-color: $color-page;
+			border: 1px solid $color-search-grid-border;
+			border-right: 2px solid $color-search-form-separator;
+
+			height: 24px;
+			padding: 3px 12px;
+			float: left;
+
+			.wds-dropdown__toggle {
+				font-weight: bold;
+			}
+
+			.wds-dropdown__content {
+				padding: 0;
+			}
+
+			.wds-list {
+				margin: 3px 11px 3px 0 !important;
```

Comment:
why there is a need of overriding margins, paddings and height of dropdown?

Reply:
I tried to match mockups - without overrides there was too much space around list and list items. I didn't use css for a while, so I guess it can be done better.